src/components/forms/ItemForm.svelte

```diff
@@ -74,44 +75,38 @@ const getFormData = () => {
     shortName,
     purchaseDate,
     uniqueIdentifier,
+    shouldSubmit
   }
 }
+
 const onCategorySelectPopulated = () => {
   if (item.category?.id) {
     initialCategoryId = item.category?.id
   }
 }
-const onSubmit = () => {
+
+const onSubmit = (event: Event) => {
+  shouldSubmit = true
   dispatch('submit', getFormData())
 }
-const saveForLater = () => {
-  coverageStatus = 'Draft'
-  // then onSubmit is automatically called by the form
+
+const saveForLater = (event: Event) => {
+  dispatch('submit', getFormData())
```

Comment:
This should be 'save-for-later'
```suggestion
  dispatch('save-for-later', getFormData())
```

Reply:
i removed that event. do you think i should add it back?